composer.json

```diff
@@ -94,7 +94,8 @@
     "openjournalsoftware/bibtex": "~0.2",
     "exercise/htmlpurifier-bundle": "~0.2",
     "ojs/export-bundle": "dev-master",
-    "willdurand/js-translation-bundle": "^2.5"
+    "willdurand/js-translation-bundle": "^2.5",
+    "contentfarm/disposable-email-list":"dev-master"
```

Comment:
It would be great if the project depended on a specific tag of `contentfarm/disposable-email-list` instead of `dev-master`.